client/src/components/atoms/TextField.tsx

```diff
@@ -8,24 +8,29 @@ interface TextFieldProps {
   name: string,
   placeholder: string,
   type: "text" | "password",
+  iconClassName?: string
 }
 
 const TextField: FunctionComponent<TextFieldProps> = (props: TextFieldProps) => {
-  return <React.Fragment>
```

Comment:
not too familiar with react fragments, but is it ok to just replace the fragment with a div here?